autoendpoint/src/routers/mod.rs

```diff
@@ -194,7 +194,7 @@ impl ReportableError for RouterError {
             | RouterError::RequestTimeout
             | RouterError::TooMuchData(_)
             | RouterError::Upstream { .. } => false,
-            RouterError::SaveDb(e) => e.is_sentry_event(),
+            RouterError::SaveDb(e, _) => e.is_sentry_event(),
```

Comment:
In this case, we dispose of it because we are testing to see if the error should be reported to sentry `is_sentry_event()`. The `ReportableError` [trait](https://github.com/mozilla-services/autopush-rs/blob/c30ba14120d9a6440bfff2b0181e4b9cee00d84f/autopush-common/src/errors.rs#L138-L139) has additional methods that provide the extra content.  For `RouterError`, the `extras` [method](https://github.com/mozilla-services/autopush-rs/blob/c30ba14120d9a6440bfff2b0181e4b9cee00d84f/autoendpoint/src/routers/mod.rs#L222-L223) provides the `SaveDb` string as extra sentry message data. This prevents sentry from getting high cardinality content, while still providing devs a cause, albeit a non-searchable one.